crates/symbolicator-service/src/download/mod.rs

```diff
@@ -152,6 +156,10 @@ impl HostDenyList {
     /// If that puts the host over the threshold, it is added
     /// to the blocked servers.
     fn register_failure(&self, source_name: &str, host: String) {
+        if self.never_block.contains(&host) {
+            return;
+        }
```

Comment:
you should maybe move this down to line 202.

We would want to count the failures and emit logs, but just avoid putting it into `blocked_hosts`.